hermes/tests.py

```diff
@@ -1,3 +0,0 @@
-from django.test import TestCase
-
-# Create your tests here.
```

Comment:
Thank you, thank you, thank you for adding tests!!! (I'm so embarrassed that we've neglected testing until now!).